analyzer/src/main/kotlin/managers/CocoaPods.kt

```diff
@@ -46,7 +86,206 @@ class CocoaPods(
         ) = CocoaPods(managerName, analysisRoot, analyzerConfig, repoConfig)
     }
 
+    private val podspecCache: MutableMap<String, Podspec> = mutableMapOf()
+
+    override fun command(workingDir: File?) = "pod"
+
+    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[1.10.1,)")
+
+    override fun getVersionArguments(): String = "--version --allow-root"
+
+    override fun beforeResolution(definitionFiles: List<File>) = checkVersion(analyzerConfig.ignoreToolVersions)
+
     override fun resolveDependencies(definitionFile: File): List<ProjectAnalyzerResult> {
-        TODO("not implemented") // To change body of created functions use File | Settings | File Templates.
+        // CocoaPods originally used and may still use the Specs repository on GitHub [1] as package meta-data database.
+        // Using [1] requires an initial clone which is slow to do and consumes already more than 5 GB on disk, see
+        // also [2]. (Final) CDN support has been added in version 1.7.2 [3] to speed things up.
+        //
+        // Temporarily re-configure the repos such that 'https://cdn.cocoapods.org' is the only repository in order to
+        // avoid having to clone and fetch the large Git repository. This ensures that the repositories used are
+        // independent of the hosts current setup.
+        //
+        // [1] https://github.com/CocoaPods/Specs
+        // [2] https://github.com/CocoaPods/CocoaPods/issues/7046.
+        // [3] https://blog.cocoapods.org/CocoaPods-1.7.2/
+
+        stashDirectories(File("~/.cocoapods/repos")).use {
+            run("repo", "add-cdn", "trunk", "https://cdn.cocoapods.org")
+
+            try {
+                return resolveDependenciesInternal(definitionFile)
```

Comment:
Nit: Move out `return` to before `stashDirectories`?

Reply:
You moved this to before `try`, not to before `stashDirectories`. Was this intended?

Reply:
I intended to do what you suggested, so I did wrong. Moved it to before `stashDirectories`.